src/ducks/modules/protocol/utils/dialogs.js

```diff
@@ -10,18 +11,23 @@ export const invalidAssetErrorDialog = (e, filePath) => {
         Canvas.
       </p>
       <p>
-        <strong>Reminder:</strong> Network Canvas supports importing network data in
-        either JSON or CSV format. JSON files must contain a &quot;nodes&quot; and/or
-        &quot;edges&quot; list in the root object, and CSV data should contain a header
-        row comprising variable names, and be in the format of a node or edge list.
-      </p>
-      <p>
-        Variable names must use only letters, numbers, the symbols ._-: and contain no spaces.
+        The file you attempted to import is not in a format supported by
```

Comment:
This ends up repeating the same sentence twice, right? "The file you attempted to import is not in a format supported by Network Canvas." Is that intentional?

Reply:
:facepalm:

Reply:
Have removed the duplicate line.